src/process/id.rs

```diff
@@ -113,6 +113,9 @@ pub fn getpid() -> Pid {
 
 /// `getppid()`—Returns the parent process' ID.
 ///
+/// This will return `None` if the parent process ID is 0; this will happen, for instance, for an
+/// init process (PID 1).
```

Comment:
According to [the docs](https://man7.org/linux/man-pages/man2/getpid.2.html#NOTES), `getppid` returns 0 if the parent is in a different pid namespace.

And as a minor nit, no process ever has ID 0; 0 is a sentinel value rather than an ID value.

What would you think of wording this as "This will return `None` if the the process is the init process (PID 1) or if the parent is in a different PID namespace"?

Reply:
My goal with saying "for instance" here was to avoid attempting to *exhaustively* enumerate the circumstances that could lead to this, not least of which because those circumstances are OS-specific. However, I can clarify this a bit further.